hubblestack_nova_profiles/cis/ubuntu-1604-level-1-scored-v1-0-0.yaml

```diff
@@ -1,566 +1,752 @@
-# This is mostly a copy from the 14.04 profile, until we have time to do
-# a proper profile
-
 grep:
   whitelist:
+    disable_mount_cramfs:
+      data:
+        Ubuntu-16.04:
+        - /etc/modprobe.d:
```

Comment:
Yes but i have checked that we can temporarily disable the module checking. That's why written in this file. if somebody will try to enable to particular FS then he/she will not be able to do it.

Reply:
@robinku07 Can you elaborate what you mean by this? Is the intention to allow someone to temporarily enable a particular FS?  Hubble does nothing to prevent that; it only reports it as a failed check.  /proc/* checks prevent false negatives that an attacker can induce.  This is important for security.  You can always allow exceptions in an operation to satisfy temporary needs without compromising fidelity of data.

Reply:
@whackyhack is correct. Let's handle exceptions further down the line. Hubble should be checking `/proc/modules` to see the current state of the system and reporting on that.